uniffi_bindgen/src/bindings/kotlin/templates/NamespaceLibraryTemplate.kt

```diff
@@ -55,20 +55,65 @@ internal open class {{ ffi_struct.name()|ffi_struct_name }}(
 {%- endmatch %}
 {%- endfor %}
 
+
+{%- macro decl_kotlin_functions(func_list) -%}
+{% for func in func_list -%}
+fun {{ func.name() }}(
+    {%- call kt::arg_list_ffi_decl(func) %}
+): {% match func.return_type() %}{% when Some with (return_type) %}{{ return_type.borrow()|ffi_type_name_by_value }}{% when None %}Unit{% endmatch %}
+{% endfor %}
+{%- endmacro %}
+
+// For large crates we prevent `MethodTooLargeException` (see #2340)
```

Comment:
let me know if you want me to make those comments not be part of emitted Kotlin code, but "local to" the template (Rinja comments), personally I like having them in the generated kotlin code. Let me know!